Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -785,4 +794,24 @@ open class _ComposerVC<ExtraData: ExtraDataTypes>: _ViewController,
         alert.addAction(.init(title: L10n.Alert.Actions.ok, style: .default, handler: { _ in }))
         present(alert, animated: true)
     }
+    
+    // MARK: - InputTextViewImagePasteDelegate
+    
+    open func inputTextView(_ inputTextView: InputTextView, didPasteImage image: UIImage) {
+        do {
+            let imageUrl = try image.temporaryLocalFileUrl()
+            let urlAndType: (URL, AttachmentType) = (imageUrl!, .image)
+            let fileSize = try AttachmentFile(url: urlAndType.0).size
+            let maxFileSize = channelController?.client.config.maxAttachmentSize ?? 0
+            
+            if fileSize < maxFileSize {
+                let attachment = try AnyAttachmentPayload(localFileURL: urlAndType.0, attachmentType: urlAndType.1)
+                content.attachments.append(attachment)
+            } else {
+                showAttachmentExceedsMaxSizeAlert()
+            }
```

Comment:
the same attachment size validation logic is used for image picker selection. Does it makes sense to encapsulate it in some func and reuse?

Reply:
Added a common function